### ISSUE TITLE
The result on ResNet18 is frustrating

### ISSUE THREAD

**Issue by iamhankai** [kakaobrain/fast-autoaugment#42]
Hi Ildoo, thank you for sharing great code.

I tried ResNet18 on ImageNet, but the result is not good. Have you ever experimented on ResNet18, or do you have any suggestions?

Hyperparameters: SGD with linear lr=0.4, batch=1024, weight decay=1e-4, epochs=120.

|method|top1|
|-|-|
|baseline|70.68|
|fast aa|70.22|

The learning curves:
![image](https://user-images.githubusercontent.com/9500784/78129843-ccc3b000-744a-11ea-91bd-69955b2830c2.png)

**Comment by ildoonet** [comment]
@iamhankai 

There is a bug with learning rate scheduler with recent version of pytorch. I fixed it so you can try again. 

Also I will look into this problem as soon as I can.

https://github.com/kakaobrain/fast-autoaugment/commit/2424224291616a2158a2265069ee95a78a2d13d9
